src/components/views/TeamDashboard.tsx

```diff
@@ -41,16 +41,16 @@ const TeamDashboard: React.FC = () => {
     const channel = pusher.subscribe(`team-${teamId}`);
     channel.bind("session-update", (data: { status: string }) => {
       setSessionStatus(data.status);
-      window.location.reload();
+      fetchStatus();
     });
 
     channel.bind("task-update", () => {
-      window.location.reload();
+      fetchTeamTasks();
     });
```

Comment:
please add after fetchTeamMembers to make VoiceChat work:
document.dispatchEvent(new CustomEvent('checkBoxChange'))